xmipp

```diff
@@ -754,42 +760,43 @@ def checkMatlab(configDict):
         runJob("rm xmipp_mex*")
     return ans
 
+
 def writeConfig(configDict):
-    with open("xmipp.conf", "w") as configFile:
+    with open(CONFIG_FILE_NAME, "w") as configFile:
         configFile.write("[BUILD]\n")
         for label in sorted(configDict.keys()):
             configFile.write("%s=%s\n"%(label,configDict[label]))
 
+
+def config_DL(configDict):
+    k = 'USE_DL'
+    if (k in configDict) and (configDict[k] != 'False'):
+        configDict[k] = 'True'
+
+
 def config():
     print("Configuring -----------------------------------------")
-    configDict = createEmptyConfig()
-    updateConfigDict(configDict, readConfigFile("xmipp.conf"))
-    scipionHomeDir = getScipionHome()
+    new_config_dict = createEmptyConfig()
     useScipion = not ('XMIPP_NOSCIPION' in os.environ)
     if useScipion and not scipionHomeDir:
```

Comment:
This is failing to me since scipionHomeDir is not defined when I run ./xmipp.

I change this for `if useScipion and not getScipionHome():` since I'm working on the download deep learning models in this branch.

Reply:
Sounds good